src/wrapper/index.jsx

```diff
@@ -1,9 +1,9 @@
 import React, { Fragment } from 'react';
-import { Alert } from '../';
+import { Notifiction } from '../';
```

Comment:
Typo.

Reply:
another reason we should get rid of the build - no errors are currently thrown on compile